src/webportal/src/app/job/job-view/fabric/job-retry/job-retry-card.jsx

```diff
@@ -0,0 +1,313 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import { FontClassNames, ColorClassNames, getTheme } from '@uifabric/styling';
+import c from 'classnames';
+import { Stack, IconButton, Link } from 'office-ui-fabric-react';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Interval, DateTime } from 'luxon';
+import { capitalize, isNil, get } from 'lodash';
+import styled from 'styled-components';
+import yaml from 'js-yaml';
+
+import { getDurationString } from '../../../../components/util/job';
+import StatusBadge from '../../../../components/status-badge';
+import { ContainerList } from './container-list';
+import { printDateTime } from '../job-detail/util';
+import MonacoPanel from '../../../../components/monaco-panel';
+
+const { spacing, palette } = getTheme();
+
+function getAttemptDurationString(attempt) {
+  const start =
+    attempt.attemptStartedTime &&
+    DateTime.fromMillis(attempt.attemptStartedTime);
+  const end = attempt.attemptCompletedTime
+    ? DateTime.fromMillis(attempt.attemptCompletedTime)
+    : DateTime.utc();
+  if (start && end) {
+    return getDurationString(
+      Interval.fromDateTimes(start, end || DateTime.utc()).toDuration([
+        'days',
+        'hours',
+        'minutes',
+        'seconds',
+      ]),
+    );
+  } else {
+    return 'N/A';
+  }
+}
+
+const RetryCard = styled.div`
+  background: #f8f8f8;
+  box-shadow: rgba(0, 0, 0, 0.06) 0px 2px 4px, rgba(0, 0, 0, 0.05) 0px 0.5px 1px;
+`;
+
+const TaskRoleCard = styled.div`
```

Comment:
The Card component has style and classname props, you can add your custom style there.
If the style is not reused, I think you don't need to create a new component with copied code here.